scene/resources/bit_mask.cpp

```diff
@@ -168,10 +169,351 @@ Dictionary BitMap::_get_data() const {
 	return d;
 }
 
+Vector<Vector2> BitMap::_march_square(const Rect2i &rect, const Point2i &start) const {
+
+	int stepx = 0;
+	int stepy = 0;
+	int prevx = 0;
+	int prevy = 0;
+	int startx = start.x;
+	int starty = start.y;
+	int curx = startx;
+	int cury = starty;
+	unsigned int count = 0;
+	Set<Point2i> case9s;
+	Set<Point2i> case6s;
+	int i;
+	Vector<Vector2> _points;
+	do {
+		int sv = 0;
+		{ //square value
+
+			/*
+	     checking the 2x2 pixel grid, assigning these values to each pixel, if not transparent
```

Comment:
Indentation seems to use spaces here?

Reply:
Yes. I've checked and it's the same in _master_: https://github.com/godotengine/godot/blob/master/scene/resources/bit_mask.cpp

And clang-format isn't doing anything about it. It seems it's keeping spaces within comments beyond the code nesting level.

Reply:
Now fixed in master :) ccf5a5025

Reply:
And now here too!